src/engine/Input/Pointer.ts

```diff
@@ -80,6 +86,44 @@ export class PointerEvent extends GameEvent<any> {
 };
 
 /**
+ * Wheel Events
+ *
+ * Represents a mouse wheel event. See [[Pointers]] for more information on
+ * handling point input.
+ */
+export class WheelEvent extends GameEvent<any> {
```

Comment:
Maybe? Would we use an Enum for that? WheelDirection { Up, Down, Left, Right, In, Out }?

Theoretically, you could have *+ or -* in *x, y, or z*. Although in most circumstances, only *y* will be used (as thats the typical scroll wheel).